db/controllers/review-controller.js

```diff
@@ -8,36 +8,36 @@ const reviewRouter = express.Router();
 reviewRouter.post("/", async (req, res) => {
     // Validate request
     if (!req.body.userId && !req.body.albumId && !req.body.score) {
-      res.status(400).send({
-        message: "Fields can not be empty!",
-      });
-      return;
+        res.status(400).send({
+            message: "Fields can not be empty!",
+        });
+        return;
     }
-  
+
     try {
-      // Update or create the review in the database
-      const [review, created] = await Review.upsert(
-        {
-          UserId: req.body.userId,
-          AlbumId: req.body.albumId,
-          score: req.body.score,
-          reviewText: req.body.reviewText,
-        },
-        { returning: true } // Return the updated review object
-      );
-  
-      res.send(review);
+        // Update or create the review in the database
+        const [review, created] = await Review.upsert(
+            {
+                UserId: req.body.userId,
+                AlbumId: req.body.albumId,
+                score: req.body.score,
+                reviewText: req.body.reviewText,
+            },
+            {returning: true} // Return the updated review object
+        );
+
+        res.send(review);
     } catch (err) {
-      console.error(err);
-      res.status(500).send({
-        message: "Some error occurred while creating or updating the Review.",
-      });
+        console.error(err);
+        res.status(500).send({
+            message: "Some error occurred while creating or updating the Review.",
+        });
     }
 });
 
 reviewRouter.get("/", (req, res) => {
     // Get all reviews
-    Review.findAll()
+    Review.findAll({include: [User, Album]})
```

Comment:
Needed to include the User and Album object to have access to the necessary info to display the review on the home page

Reply:
This makes sense